media/css/firefox/tiles.less

```diff
@@ -4,10 +4,136 @@
 
 @import "../sandstone/lib.less";
 
+#outer-wrapper {
+    overflow: hidden;
+}
+
 #wrapper {
     margin-top: 50px;
 }
 
-.sidebar .sidebar-inset img {
-    border: none;
+.sidebar {
+    float: left;
+    margin: -200px 10px 0 10px;
+
+    .sidebar-inset {
+        background-color: transparent;
+        box-shadow: none;
+        border: none;
+        padding: 0;
+
+        img {
+            border: none;
+            max-width: none;
+            margin-bottom: @gridGutterWidth;
+        }
+
+        img.tab {
+            width: 200%;
+        }
+
+        img.tab-preferences {
+            width: 100%;
+        }
+    }
+}
+
+.main-column {
+    width: 580px;
+
+    .intro {
+        font-size: @baseLine;
+        font-size: 1.2rem;
+        font-style: italic;
+        font-weight: 100;
+        padding-bottom: 2em;
+    }
+
+
+    section {
+        padding-top: 2.5em;
+        border-top: solid 3px @borderColor;
+
+        h4 {
+            font-weight: bold;
+            font-style: italic;
```

Comment:
This is a bit of a tricky one. Because you are inheriting `Open Sans Light` from `sandstone.less` here, what you are actually getting in the browser is faux bold but, we are also not currently downloading `Open Sans Bold Italic` so, the italic will also be faux. The italic seems the lesser of the two evils ;) though so, I would suggest adding:

```
font-family: "Open Sans";
```
If you feel that the faux italic does not look great, you can add the required font to this page: https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/fonts.less#L66

Perhaps @alexgibson has some thoughts on this as well?